inappmessaging/build.gradle

```diff
@@ -55,14 +55,14 @@ dependencies {
 
   implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
   implementation 'androidx.work:work-runtime:2.7.0'
-  implementation 'com.facebook.fresco:animated-gif:2.5.0'
-  implementation 'com.facebook.fresco:fresco:2.5.0'
   implementation 'com.google.android.material:material:1.2.1'
   implementation 'com.jakewharton.timber:timber:4.7.1'
   implementation 'com.squareup.retrofit2:converter-gson:2.5.0'
   implementation 'com.squareup.retrofit2:retrofit:2.8.1'
   implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
   implementation 'com.google.code.gson:gson:2.8.6'
+  implementation "io.github.rakutentech.manifestconfig:manifest-config-annotations:0.2.0"
```

Comment:
This should be removed (from Android 12 support)

Reply:
removed